proxy/operator_resource/resource_manager.py

```diff
@@ -121,6 +129,53 @@ def free_resource(self, is_good_resource: bool, op_resource: OpResourceModel) ->
                 _LOG.debug("disable resource: %s", op_resource)
                 op_signer.disabled_holder_list.append(op_holder)
 
+            try:
+                self._stat_client.commit_op_resource_holder_status(
+                    OpResourceHolderStatusData(
+                        owner=op_resource.owner,
+                        free_holder_cnt=len(op_signer.free_holder_list),
+                        used_holder_cnt=len(op_signer.used_holder_dict),
+                        disabled_holder_cnt=len(op_signer.disabled_holder_list),
+                    )
+                )
```

Comment:
The number of holders is also changed:
- on activating signer keys, when holders are created.
- on destroying the blocked holders.

Reply:
I wasn't too sure about that one since I couldn't find how they were blocked, but thanks for confirming. Done.

Reply:
Ah, ok. Will add to bottom of `_refresh_signer_list()`

Reply:
You mentioned factoring out the holder stat call and add it to one place, but stepping through the code I think it should be called from within all of the `*_list()` functions called from ` _activate_signer_loop(self)`. But we don't want to call it from `_activate_signer_loop(self)` itself because we should only call it when something changes.

Maybe I'm wrong here. Could you please take a look.